### ISSUE TITLE
How to run on SimiFeat custom datasets ?

### ISSUE THREAD

**Issue by wengshihao** [UCSC-REAL/SimiFeat#3]
I would like to know how to run SimiFeat on custom datasets, the repository code seems very specific to datasets, have you made SimiFeat into a tool?

Thanks

**Comment by zwzhu-d** [comment]
Yes. We have integrated SimiFeat into Docta. Please refer to [this repo](https://github.com/Docta-ai/docta).
